src/translations/fr.json

```diff
@@ -68,5 +68,10 @@
     "profiles.form.modification.updateError": "Erreur lors de la modification du profil",
 
     "linked.path.display.invalidLink": "lien vers paramètres invalide.",
-    "linked.path.display.noLink": "pas de paramètres selectionnés."
+    "linked.path.display.noLink": "pas de paramètres selectionnés.",
+
+    "users.form.delete.title": "Supprimer utilisateur",
+    "users.form.delete.multipleItemsDialogMessage": "{itemsCount} utilisateurs vont être supprimés.",
+    "users.form.delete.itemDialogMessage": "{itemName} va être supprimé.",
+    "users.form.delete.multipleItemsDialogTitle": "Supprimer utilisateurs"
```

Comment:
the dialog title in french is different : missing {itemsCount}

Reply:
Fixed